packages/flutter_tools/lib/src/project.dart

```diff
@@ -84,11 +84,11 @@ class FlutterProject {
 
   /// Returns a [FlutterProject] view of the current directory or a ToolExit error,
   /// if `pubspec.yaml` or `example/pubspec.yaml` is invalid.
-  static FlutterProject current() => fromDirectory(globals.fs.currentDirectory);
+  static FlutterProject current() => globals.projectFactory.fromDirectory(globals.fs.currentDirectory);
```

Comment:
what is this change fixing?

Reply:
It doesn't fix anything, but I thought that it was a little clearer to me to just be explicit since `fromDirectory()` is simply the same thing as `globals.projectFactory.fromDirectory()`.

Reply:
ahh, right